amqprs/src/api/tls.rs

```diff
@@ -134,9 +157,15 @@ impl TlsAdaptor {
         loop {
             match rustls_pemfile::read_one(rd)? {
                 None => return Ok(keys),
-                Some(rustls_pemfile::Item::RSAKey(key)) => keys.push(key), //PKCS1
-                Some(rustls_pemfile::Item::PKCS8Key(key)) => keys.push(key),
-                Some(rustls_pemfile::Item::ECKey(key)) => keys.push(key), //SEC1
+                Some(rustls_pemfile::Item::Pkcs1Key(key)) => {
```

Comment:
Used https://github.com/rustls/pemfile/releases/tag/v%2F2.0.0 to update the variants